griptape/tasks/image_query_task.py

```diff
@@ -68,8 +69,9 @@ def image_query_engine(self, value: ImageQueryEngine) -> None:
         self._image_query_engine = value
 
     def run(self) -> TextArtifact:
-        query, image_artifacts = self.input
+        query = self.input.value[0]
+        image_artifacts = [input for input in self.input.value[1:] if isinstance(input, ImageArtifact)]
```

Comment:
This quietly drops non-image inputs, should we raise an exception if `self.input.value[1:]` aren't all `ImageArtifact`s like in the in/outpainting tasks?